scripts/build_sasl.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Current convention accross different repos is to have a `ci_scripts` folder for ci scripts.